docs/source/user-guide/troubleshooting.rst

```diff
@@ -6,17 +6,114 @@ Troubleshooting
    :local:
    :depth: 1
 
+Numpy MKL library load failed
+=============================
+
+Error messages like
+
+    Intel MKL FATAL ERROR: Cannot load mkl_intel_thread.dll
+
+or
+
+    The ordinal 241 could not be located in the the dynamic link library
+
+Cause
+-----
+
+Numpy is unable to load the correct MKL or intel openmp runtime libraries. This
+is almost always caused by one of two things:
+
+  1. the environment with numpy has not been activated
+  2. another software vendor has installed MKL or intel openmp (libiomp5md.dll)
+     files into the C:\Windows\System32 folder. These files are being loaded
+     before Anaconda's, and they're not compatible.
+
+Solution
+--------
+
+If you are not activating your environments, start with doing that. There's more
+info at the activate-env_ page. If you are still stuck, you need to consider
+more drastic measures.
+
+  1. remove any mkl-related files from C:\Windows\System32. We recommend
+     renaming them to add .bak to the filename to effectively hide them. Observe
+     if any other software breaks. Try moving the DLL files alongside the exe of
+     the software that broke. If it works again, you can keep things in the
+     moved state - Anaconda doesn't need MKL in System32, and nothing should. If
+     you identify software that is installing software here, please contact the
+     creators of that software and inform them that their practice of installing
+     MKL to a global location is fragile and is breaking other people's software
+     and wasting a lot of time.  See the list of guilty parties below.
+  2. You may try a special DLL loading mode that Anaconda builds into Python.
+     This changes the DLL search path from system32 first to system32 as another
+     entry on PATH, allowing libraries in your conda environment to be found
+     before the libraries in system32. Control of this feature is done with
+     environment variables. Only python builds beyond these builds will react to
+     these environment variables:
+
+       * python 2.7.15 build 14
+       * python 3.6.8 build 7
+       * python 3.7.2 build 8
+
+     Control environment variables:
+
+       * `CONDA_DLL_SEARCH_MODIFICATION_ENABLE`
+       * `CONDA_DLL_SEARCH_MODIFICATION_DEBUG`
+       * `CONDA_DLL_SEARCH_MODIFICATION_NEVER_ADD_WINDOWS_DIRECTORY`
+       * `CONDA_DLL_SEARCH_MODIFICATION_NEVER_ADD_CWD`
+
```

Comment:
I'd recommend specifying an example conda command for updating to that build or later and where they should set those control environment variables and to what.  Reading this, I think I could update python appropriately, but I don't know what to do with the control environment variable information.  (and some people running into this problem may be even more newbie than me)